build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -387,6 +390,10 @@ antreaProxy:
   # Note that this option is experimental. If kube-proxy is removed, option kubeAPIServerOverride must be used to access
   # apiserver directly.
   proxyAll: {{ .proxyAll }}
+  # hostNetworkMode tells antrea-agent to proxy all Service traffic in the Node's host network using either nftables or
+  # iptables. The default value is "iptables". If "nftables" is specified but nftables is unavailable on the Node, or the
```

Comment:
Actually in this case, if the user explicitly specifies nftables but it is not available for the Node, I think we should fail agent initialization.

Reply:
My bad. I forgot updating this. The implementation is just what you are expecting. We should fail the agent if this option is set to nftables but nftables is not supported in the Node.